tests/e2e/support/objects/app-files/link/index.ts

```diff
@@ -16,4 +16,15 @@ export class Link {
     linkStore.set(args.name, { name: args.name, url, password: args.password })
     await this.#page.goto(startUrl)
   }
+
+  async edit(args: Omit<editLinkArgs, 'page'>): Promise<void> {
+    const startUrl = this.#page.url()
+    let name = args.oldName
+    const url = await editLink({ page: this.#page, ...args })
+    if (args.newName) {
+      name = args.newName
+    }
+    linkStore.set(name, { name: name, url, password: args.password })
+    await this.#page.goto(startUrl)
```

Comment:
why not just use `args.newName` here, then we can skip all the boilerplate above.